presto-main/src/main/java/com/facebook/presto/server/ResourceGroupStateInfoResource.java

```diff
@@ -51,15 +53,22 @@ public ResourceGroupStateInfoResource(ResourceGroupManager<?> resourceGroupManag
     @Produces(MediaType.APPLICATION_JSON)
     @Encoded
     @Path("{resourceGroupId: .+}")
-    public ResourceGroupInfo getQueryStateInfos(@PathParam("resourceGroupId") String resourceGroupIdString)
+    public ResourceGroupInfo getQueryStateInfos(
+            @PathParam("resourceGroupId") String resourceGroupIdString,
+            @QueryParam("includeQueryInfo") @DefaultValue("true") boolean includeQueryInfo,
+            @QueryParam("summarizeSubgroups") @DefaultValue("true") boolean summarizeSubgroups,
+            @QueryParam("includeDynamicSubgroups") @DefaultValue("true") boolean includeDynamicSubgroups)
```

Comment:
may be call this as `includeStaticSubgroupsOnly` to make it consistent with how we tag the resource group.